teams/compiler-contributors.toml

```diff
@@ -12,6 +13,7 @@ members = [
   "nikic",
   "nnethercote",
   "scalexm",
+  "Mark-Simulacrum",
```

Comment:
Seems like your intention was to alphabetically sort names, but given that's not simulacrum but Mark-Simulacrum should be placed a bit above :smile:

Reply:
ha thanks, I originally had him in there "simulacrum"...

Reply:
Yeah, I guessed that that was the issue :).